api/src/kots_app/kots_ffi.ts

```diff
@@ -117,6 +117,7 @@ export async function kotsAppDownloadUpdates(updatesAvailable: Update[], app: Ko
       await kotsAppDownloadUpdate(update.cursor, app, registryInfo, stores);
     } catch (err) {
       console.error(`Failed to download release ${update.cursor}: ${err}`);
+      throw err;
```

Comment:
Does this need to be ReplicatedError type so it can be handled on by the frontend?

Reply:
Throwing just the error has been working well. I had it as a replicated error but it just added more text and almost made it worse...